### PR TITLE
Braian.quinteros/test

### PR DESCRIPTION
**Pull request by BQuinteros**
Added test to method onNumberPressed and coverage its ok.
Test

### CODE REVIEW

**Comment by BQuinteros** [comment]
Added CleanAllTest and modified onNumberPressedTest. Coverage is okey.
ClearAllTest
onNumberPressedTest

**Comment by regmarcos** [comment]
Hi! Now the CalculatorPresenter class has 100% coverage.

Anotación 2019-11-13 132150

File: app/src/test/java/com/example/calculadoramvp/ExampleUnitTest.java [hunk 1]
```diff
@@ -1,14 +1,14 @@
 package com.example.calculadoramvp;
 
 import org.junit.Test;
-
 import static org.junit.Assert.*;
```
Comment (SartiMau) [inline app/src/test/java/com/example/calculadoramvp/ExampleUnitTest.java]: not use wildcard imports pls

File: app/src/test/java/com/example/calculadoramvp/mvp/model/presenter/PresenterTest.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.example.calculadoramvp.mvp.model.presenter;
+
+public class PresenterTest {
+}
```
Comment (lucianoeli) [inline app/src/test/java/com/example/calculadoramvp/mvp/model/presenter/PresenterTest.java]: Delete this class if its gonna be empty

File: app/src/test/java/com/example/calculadoramvp/mvp/model/presenter/PresenterTest.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package com.example.calculadoramvp.mvp.model.presenter;
+
+public class PresenterTest {
```
Comment (SartiMau) [inline app/src/test/java/com/example/calculadoramvp/mvp/model/presenter/PresenterTest.java]: if this is not needed pls remove it

**Review by SartiMau — CHANGES_REQUESTED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**